src/content/docs/en/guides/middleware.mdx

```diff
@@ -46,22 +47,23 @@ This is an optional argument passed to `onRequest()` that may contain the `local
 
 ### Storing data in `context.locals`
 
-`context.locals` is an object containing data from a `Response` that can be manipulated inside middleware. 
+`context.locals` is an object that can be manipulated inside the middleware. Adapters _might_ add data to it.
```

Comment:
"Adapters might add data to it" is a little too vague to be helpful.

It's not clear whether the reader should take this as some kind of warning (i.e. if you're writing your own middleware , beware that there might be something else that's also manipulating the data in `context.locals`), or it's being used as an example of something that manipulates `context.locals` to provide real-usage context, or something else entirely.... fun fact!  Did you know...! :smile: 

Can you reword that statement in a way that makes it clear *why* they are being told that?

Reply:
I think this could be added on a tip. What do you think of removing that last sentence and adding this:

```md
:::tip[Adapter-provided properties]
Adapters may provide extra runtime information and functionality through the `locals` object. For example, SSR adapters may include information about the request or runtime.
:::
```

We can link to some examples. Of the official integrations, 4 modify the `locals` object.

Reply:
I still don't know **why** you're telling me this, though? Again, it's not clear why I should care about this, whether it's something for me to watch out for...

Reply:
I think it is a mix of everything. It is something to be aware:
- An integration may provide something that you are checking manually
- Something you write might override a value used by an integration
- Information that would be hard to access or infer otherwise might already be included there (like CloudFlare's runtime)

Reply:
OK, gotcha! No, I don't want examples here to clutter this up. The reasoning/why I should care is way more important. I think what you mentioned above is the kind of content that would be good to go inside a note!

Reply:
I sent a new version with a different tip. WDYT?